charts/sn-platform/templates/vault/vault-initialize-public-key.yaml

```diff
@@ -50,7 +50,7 @@ spec:
            echo "vault root token is ready";
            rootToken=$(kubectl get secret {{ template "pulsar.vault-unseal-secret-key-name" . }} -o jsonpath='{.data.vault-root}'|base64 -d);
            echo $rootToken;
-           echo $rootToken > /root/rootToken;
+           echo $rootToken > /vault/rootToken;
```

Comment:
How will `/vault/rootToken` be used?

Reply:
non-root users can't access /root directory, and from vault Dockerfile https://github.com/hashicorp/docker-vault/blob/cba1f2209c3bddb1319c56b17ef308a2c6cb6c7c/ubi/Dockerfile#L66 we know vault has a home directory for user vault

Reply:
@fantapsody PTAL